browser/src/control/Control.JSDialog.js

```diff
@@ -710,8 +710,10 @@ L.Control.JSDialog = L.Control.extend({
 				instance.updatePos();
 			}
 
-			if (instance.isAutofilter)
+			if (instance.isAutofilter && instance.id !== 'autoFillPreviewTooltip')
```

Comment:
pleae do not use hardcoded ids

Reply:
We enter the autofilter case there.

Reply:
I see `isAutofilter` is too general now: `instance.isAutofilter = instance.isDocumentAreaPopup && this.map._docLayer.isCalc();` 

Can we check if `instance.isAutoCompletePopup` can be used here?

Reply:
I've used `instance.isAutoFillPreviewTooltip` here. 
Done.